pcsx2/gui/ConsoleLogger.cpp

```diff
@@ -514,6 +548,11 @@ ConsoleLogFrame::ConsoleLogFrame( MainEmuFrame *parent, const wxString& title, A
 	{
 		pMenuBar->Check(MenuId_ColorScheme_Light, true);
 	}
+	else //if ((0==m_conf.Theme.CmpNoCase("Default")) || (0==m_conf.Theme.CmpNoCase("ClassicDark")))
```

Comment:
Need to change the above statement to else if (0==m_conf.Theme.CmpNoCase("Light")))